src/FunFair.CodeAnalysis.Tests/FunFair.CodeAnalysis.Tests.csproj

```diff
@@ -32,6 +32,7 @@
     <PackageReference Include="SourceLink.Create.CommandLine" Version="2.8.3" PrivateAssets="All" />
     <PackageReference Include="SourceLink.Create.CommandLine" Version="2.8.3" PrivateAssets="All" />
     <PackageReference Include="xunit.analyzers" Version="0.10.0" PrivateAssets="All" />
+    <PackageReference Include="Microsoft.VisualStudio.Threading.Analyzers" Version="16.5.132" PrivateAssets="all" />
```

Comment:
Looks like its complaining about the same missing reference in FunFair.CodeAnalysis.csproj too